docs/UserGuide.md

```diff
@@ -309,15 +279,15 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
-**View** | `view n/name` <br> `view id/ID`
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/Tag`
-**Delete** | `delete n/name` <br> `delete id/ID` <br> `delete t/group`
-**Edit** | `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]` <br> `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]` <br> `edit ID [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]`
+**View** | `view n/name` <br> `view i/index`
+**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [r/ROLE] [s/STATUS]`
```

Comment:
Same for these.

Reply:
Would keeping the square brackets be better? For the table view. Just noticed we haven't standardised.

Reply:
hmm square brackets are for fields that are meant to be optional